wrappers/pkcs11/options.go

```diff
@@ -0,0 +1,167 @@
+// Copyright (c) HashiCorp, Inc.
```

Comment:
@glatigny Regarding copyright statements, if this is net-original code, I'd suggest a header of:

```go
// Copyright (c) 2024 OpenBao a Series of LF Projects, LLC
```

Many of these options &c are similar to existing patterns, so I'm happy with a mix of both headers as is likely the case.